spec/features/account_manager/duplicating_an_entry_spec.rb

```diff
@@ -1,15 +1,14 @@
 require 'spec_helper'
 
-feature 'account manager duplicating an entry' do
-  scenario 'from own account' do
-    account = create(:account)
-    entry = create(:entry, :title => 'Test Entry', :account => account)
+feature 'duplicating an entry' do
+  scenario 'as publisher of entry' do
+    user = Dom::Admin::Page.sign_in_as(:editor)
```

Comment:
For cases like this a simple `Dom::Admin::Page.sign_in` method would come in handy. Editor's used to be the unprivileged ones, but right now we really have not good way to create a unprivileged user.